.prettierrc.js

```diff
@@ -1,8 +1,9 @@
 module.exports = {
```

Comment:
Can we keep this file please as it's for this repo, not projects being migrated.